src/SOS/Strike/strike.cpp

```diff
@@ -7587,6 +7584,38 @@ class CNotification : public IXCLRDataExceptionNotification5
      */
     STDMETHODIMP OnCodeGenerated(IXCLRDataMethodInstance* method)
     {
+#ifndef FEATURE_PAL
+        if (g_isDesktopRuntime)
+        {
+            // Some method has been generated, make a breakpoint and remove it.
+            ULONG32 len = mdNameLen;
+            LPWSTR szModuleName = (LPWSTR)alloca(mdNameLen * sizeof(WCHAR));
+            if (method->GetName(0, mdNameLen, &len, g_mdName) == S_OK)
+            {
+                ToRelease<IXCLRDataModule> pMod;
+                HRESULT hr = method->GetTokenAndScope(NULL, &pMod);
+                if (SUCCEEDED(hr))
+                {
+                    len = mdNameLen;
+                    if (pMod->GetName(mdNameLen, &len, szModuleName) == S_OK)
+                    {
+                        ExtOut("JITTED %S!%S\n", szModuleName, g_mdName);
+
+                        // Add breakpoint, perhaps delete pending breakpoint
+                        DacpGetModuleAddress dgma;
+                        if (SUCCEEDED(dgma.Request(pMod)))
+                        {
+                            g_bpoints.Update(TO_TADDR(dgma.ModulePtr), FALSE);
```

Comment:
Is there no possible failure case here?

Reply:
This code came from the desktop SOS source.  Even on the failures here, there really isn't anything we can do except print an error message.